index.liquid

```diff
@@ -52,18 +52,26 @@
                         <dd><p>{{group.description}}</p><dd>
                     {% endfor %}
                 </dl>
+                <dl class="general-public">
+                <dt><h3>General Public Opening Times</h3><dt>
+                    {% for opening in supermarket.generalHours %}
+                        <dt><strong><p>{{opening.day}}</p></strong><dt>
+                        <dd><p>{{opening.times}}</p><dd>
+                    {% endfor %}
+                </dl>
```

Comment:
Gonna wrap this in a conditional as Ocado and AllSupermarkets (which came in after you did this tbf) don't have generalOpening.

```suggestion
                {% if supermarket.generalHours %}
                    <dl class="general-public">
                    <dt><h3>General Public Opening Times</h3><dt>
                        {% for opening in supermarket.generalHours %}
                            <dt><strong><p>{{opening.day}}</p></strong><dt>
                            <dd><p>{{opening.times}}</p><dd>
                        {% endfor %}
                    </dl>
                {% endif %}
```